parse/getast.go

```diff
@@ -23,10 +24,11 @@ type FileSet struct {
 
 // File parses a file at the relative path
 // provided and produces a new *FileSet.
-// (No exported structs is considered an error.)
+// (No structs is considered an error.)
```

Comment:
It shouldn't say "structs," either, since we've been able to process most composite types for a while now.